src/main/java/com/staff_management_system/com/database_connection_package/DatabaseConnect.java

```diff
@@ -0,0 +1,51 @@
+package com.database_connection_package;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+
+public class DatabaseConnect {
+    /**
+     * Connection object to open port to db
+     */
+    private final ThreadLocal<Connection> aConnection = new ThreadLocal<>();
+
+    /**
+     * closes the database connection
+     */
+    public void terminate() {
+        try {
+            aConnection.get().close();
+        } catch (SQLException e) {
+            e.printStackTrace();
+        }
+    }
+
+    /**
+     * establishes the database connection
+     *
+     * @return Connection to the database
+     */
+    public Connection initialize() {
+        try {
+            Class.forName("org.postgresql.Driver"); // loads the JDBC Driver for PostGreSQL

```

Comment:
there should not be any magic numbers or magic strings (read morea bout them online), so `org.postgresql.Driver` should be a class level constant.

Reply:
I will read about it and fix it.